ShareX.UploadersLib/Forms/UploadersConfigForm.cs

```diff
@@ -2498,6 +2512,12 @@ private void txtCustomUploaderLog_LinkClicked(object sender, LinkClickedEventArg
 
         #endregion Custom Uploaders
 
+        private void label1_Click(object sender, EventArgs e)
```

Comment:
Ah, my bad I didn't catch that. I believe it's just fluff left over from the form builder.